File: src/test/java/pl/training/module3/Module3ExamplesTest.java
package pl.training.module3;

import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;

import org.junit.jupiter.api.Test;

final class Module3ExamplesTest {
    @Test
    void runsAllModuleExamples() {
        assertDoesNotThrow(() -> Module3Examples.main(new String[0]));
    }
}

File: typescript/test/module3/Module3Examples.test.ts
import { describe, expect, it } from 'vitest';
import { Module3Examples } from '../../src/module3/Module3Examples.js';

describe('Module3ExamplesTest', () => {
  it('runsAllModuleExamples', () => {
    expect(() => Module3Examples.main([])).not.toThrow();
  });
});
